azure-pipelines-ci.yml

```
@@ -18,5 +18,17 @@ jobs:
     displayName: 'Install JQ'
   - bash: |
       shopt -s globstar
-      cat **/*.json | jq '.'
-    displayName: 'Run JQ'
+      cat components/* | jq '.'
+    displayName: 'Run JQ to validate JSON files'
+  - bash: |
+        for component in components/*; do
+          manifest=$(jq --raw-output ".manifest" "${component}")
+
+          echo "Process: ${component} -> ${manifest}"
+          curl -s "${manifest}" | jq -e --raw-output '.requirements[0]'
+          if [[ "$?" != 0 ]]; then
+            echo "Useless addition, $component does not have any requirements"
```

Can we make this output a bit more friendly? 😄 

```suggestion
            echo "Useless addition, $component does not have any requirements"
```